### DISCUSSION TITLE
Could I upgrade ceph from nautilus to pacific?

### DISCUSSION THREAD

**Discussion by zeaone** [rook/rook#11502] [Q&A] [answered]
When I was upgrade ceph from 14 to 16. When mon and mgr are restarted, but osd is not restarted at this time,all pg status was unknow. Does anyone know what's going on? and how to solve this problem

**Comment by travisn** [comment, marked as answer]
You skipped directly from v14 to v16? It's recommended to not skip major versions of Ceph. Please review the [Ceph upgrade guide](https://rook.github.io/docs/rook/v1.10/Upgrade/ceph-upgrade/). 

What does `ceph status` show? Are any daemons down? Or the PGs just report as unknown?

**Reply by zeaone** [reply]
it is my ceph image problem, The problem was solved when I changed the ceph image, thanks

**Reply by anthonyeleven** [reply]
Upstream does support skipping one major release in general, though there can be subtleties.
